crates/holochain/src/core/workflow/sys_validation_workflow.rs

```diff
@@ -42,21 +43,24 @@ mod tests;
 
 #[instrument(skip(
     workspace,
+    space,
     trigger_app_validation,
     sys_validation_trigger,
     network,
     conductor_api
 ))]
 pub async fn sys_validation_workflow(
-    workspace: SysValidationWorkspace,
+    workspace: Arc<SysValidationWorkspace>,
+    space: Arc<Space>,
     trigger_app_validation: TriggerSender,
     sys_validation_trigger: TriggerSender,
     // TODO: Update HolochainP2p to reflect changes to pass through network.
     network: HolochainP2pDna,
-    conductor_api: impl CellConductorApiT,
+    conductor_api: &dyn ConductorHandleT,
```

Comment:
Some of the `conductor_api`s have changed to `conductor_handle`s do we want to do all these in this file too?

I find conductor_api very confusing, i have no idea why we have a whole crate called that... that crate doesn't seem to have anything to do with the conductor api... it's just the config file parsing stuff...

Reply:
resolved in `d9ddef050f95c210912056c23c08372f0a2f7cf6`